src/storage/service.py

```diff
@@ -254,11 +255,12 @@ async def get_unloaded_tg_memes(limit) -> list[dict[str, Any]]:
 
 
 async def get_unloaded_vk_memes() -> list[dict[str, Any]]:
-    "Returns only MemeType.IMAGE memes"
+    """Returns memes from VK, that have not been yet uploaded to Telegram."""
+
     select_query = f"""
         SELECT
             meme.id,
-            '{MemeType.IMAGE}' AS type,
+            meme.type,
```

Comment:
@ohld Not sure whether I'm fixing a bug here or this is intentional, need your feedback